app.json

```diff
@@ -16,11 +16,22 @@
     "ACKEE_PASSWORD": {
       "description": "Password which will be used to log in to Ackee",
       "required": true
+    },
+    "ACKEE_MONGODB": {
```

Comment:
Good idea putting this in. I didn't originally put this in because the add-on put in a mongo db uri env var automatically.